apps/src/code-studio/hiddenStageRedux.js

```diff
@@ -113,6 +113,22 @@ export function toggleHidden(scriptName, sectionId, stageId, hidden) {
   };
 }
 
+export function toggleHiddenScript(scriptName, sectionId, scriptId, hidden) {
+  return dispatch => {
+    dispatch(updateHiddenScript(sectionId, scriptId, hidden));
+    $.ajax({
```

Comment:
The _only_ difference between this ajax request and the one above is the data sent.  Seems like a good candidate for a small helper.